CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ endfunction()
 
 find_package(Threads MODULE REQUIRED)
 
-find_package(pegtl 3.0.0 QUIET CONFIG)
+find_package(pegtl 3.0.1 QUIET CONFIG)
```

Comment:
3.0.1 hasn't been released yet for PEGTL, but the version has already been updated in preparation for the next release. Since we want to pick up the version with the `std::filesystem` fallback logic from the sub-module instead of a 3.0.0 release package, it makes sense to bump the requirement up here too.

Users of vcpkg should see a version number of 3.0.0 in vcpkg once that port is done, but internally PEGTL will be using version 3.0.1 because I'll use a commit ID that includes the `std::filesystem` fallbacks.